L1Trigger/L1TTrackerDTC/plugins/BuildFile.xml

```diff
@@ -0,0 +1,5 @@
+<use name="L1Trigger/L1TTrackerDTC"/>
+
+<flags CXXFLAGS="-g -Wno-unused-variable -Wno-misleading-indentation"/>
```

Comment:
these flags aren't suitable for the official release

Reply:
I removed them.